src/main/java/com/inventor/email/EmailControl.java

```diff
@@ -0,0 +1,49 @@
+package com.inventor.email;
+
+import com.inventor.email.settings.EmailSettings;
+import com.inventor.email.service.ServicesE;
+import lombok.AllArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Date;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+@AllArgsConstructor
+public class EmailControl {
+    EmailSettings working;
+    ServicesE servicesE;
+
+    @GetMapping("/emails")
+    public List<Email> getEmails() {
+        return working.getAll();
+    }
+
+    @PostMapping(consumes = "application/json")
+    public void newEmail(@RequestBody List<Email> email){
+        working.addEmail(email);
+        servicesE.sendAll();
+    }
+
+    @DeleteMapping(value = "/delete/{id}", produces = "application/json")
+    public ResponseEntity<String> deleteEmail(@PathVariable int id) {
+        if (id >= working.getAll().size()) {
+            return ResponseEntity.status(404).body("Недосяжна ціль\nПошти з вказаним id не існує");
+        }
+        working.deleteEmail(id);
+        return ResponseEntity.ok(String.format("Пошта з номером: %s була видалена!", id));
+    }
+
+    @PutMapping(value = "/date/{id}", produces = "application/json")
+    public ResponseEntity<String> updateDate(@PathVariable int id,@RequestParam(value="date") @DateTimeFormat(pattern = "dd-MM-yyyy HH:mm") Date newDate){
```

Comment:
`/date` here implies it is a sub-resource of an email which I doubt it is. The mapping here should be `/api/emails/{id}`. Also, do not pass data like dates, long strings etc. as query params. You really should do it in the body. There are a couple of reasons for that:
1. URL length is limited
2. Need to deal with character encoding (although we don't have to deal with it, still it makes a URL string uglier)
3. You may be passing some sensitive data during such operations, which will be vulnerable if contained in a URL.